### PR TITLE
Origin/prashant emi calculator home page

### PR DESCRIPTION
**Pull request by PrashantBaurai**

### CODE REVIEW

File: src/main/java/pages/EmiCalculatorHomePage.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Author PRASHANT BAURAI
```
Comment (singhpritam007) [inline src/main/java/pages/EmiCalculatorHomePage.java:2]: Author Description should be after import statement and before the class

File: src/main/java/pages/EmiCalculatorHomePage.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ */
+
+package pages;
+
+import org.openqa.selenium.By;
+
```
Comment (singhpritam007) [inline src/main/java/pages/EmiCalculatorHomePage.java:8]: There should not be gap between import statement

File: src/main/java/pages/EmiCalculatorHomePage.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    /*
+     * Action methods to enter Home Loan Amount
+     * @param Home Loan Amount
+     */
+	public void enterHomeLoanAmount(String homeLoanAmount) {
+		DriverFactory.getDriver().findElement(inputHomeLoanAmount).sendKeys(homeLoanAmount);
+		}
+	
+    /*
+     * Action methods to home Loan Interest Rate
```
Comment (singhpritam007) [inline src/main/java/pages/EmiCalculatorHomePage.java]: write proper method name

File: src/main/java/pages/EmiCalculatorHomePage.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	public void enterHomeLoanTenure (String homeLoanTenure ) {
+		DriverFactory.getDriver().findElement(inputHomeLoanTenure).sendKeys(homeLoanTenure);
+		}
+	
+    /*
+     * Action methods to get Calculated Loan EMI Per Month based on user's input
+     */
+	public int getCalculatedLoanEmiPerMonth () {
+		String monthlyEmiAmount = DriverFactory.getDriver().findElement(calculatedLoanEmiPerMonth).getText();
+	    Integer.parseInt(monthlyEmiAmount);
```
Comment (singhpritam007) [inline src/main/java/pages/EmiCalculatorHomePage.java]: provide return keyword

File: src/main/java/pages/EmiCalculatorHomePage.java (hunk 1 above)
Comment (PrashantBaurai) [reply src/main/java/pages/EmiCalculatorHomePage.java:2]: go it Sir , starts working on it.

File: src/main/java/pages/EmiCalculatorHomePage.java (hunk 2 above)
Comment (PrashantBaurai) [reply src/main/java/pages/EmiCalculatorHomePage.java:8]: go it Sir , starts working on it.

File: src/main/java/pages/EmiCalculatorHomePage.java (hunk 3 above)
Comment (PrashantBaurai) [reply src/main/java/pages/EmiCalculatorHomePage.java]: Can you please elaborate ?

**Pull request merged**